packages/styles/src/core/variables/_sizes.scss

```diff
@@ -0,0 +1 @@
+$header-height: 52px;
```

Comment:
I think this kind of variables should have a name equal to its component/element, in this case `$toolbar-height`